packages/frontend/src/pages/CreateProjectSettings.tsx

```diff
@@ -1,6 +1,7 @@
-import React, { FC } from 'react';
+import { FC } from 'react';
 import { useQueryClient } from 'react-query';
 import { useHistory, useParams } from 'react-router-dom';
+
```

Comment:
do you like to have this new line separation between modules and non-module imports?